### DISCUSSION TITLE
kafka_connect_worker_connector_count not showing on prometheus

### DISCUSSION THREAD

**Discussion by yogeshgoel001** [strimzi/strimzi-kafka-operator#10036] [General]
kafka_connect_worker_connector_count not showing on prometheus even after implementing kafka-connect-metrics.yaml

**Comment by scholzj** [comment]
Seems to work for me:

```
$ curl -s localhost:9404/metrics | grep kafka_connect_worker_connector_count
# HELP kafka_connect_worker_connector_count Kafka Connect JMX metric worker
# TYPE kafka_connect_worker_connector_count gauge
kafka_connect_worker_connector_count 3.0
```

**Reply by yogeshgoel001** [reply]
any specific setting you did to accomplish this. I am not able to see this even after implementing kafka-connect

**Comment by yogeshgoel001** [comment]
Do I need to setup prometheus in default AKS namespace or can I install in different namespace as well, as I can see it doesn't have any kafka queries.

**Reply by scholzj** [reply]
That is up to you and your environment how you setup Prometheus. But you should do what I did and check the metric in the pod first to see if the problem is there or in how your Prometheus scrapes the metric.

**Reply by yogeshgoel001** [reply]
Thanks, I can see this metrics in kafkamirrormaker2 pod. 
curl -s localhost:9404/metrics | grep kafka_connect_worker_connector_count
# HELP kafka_connect_worker_connector_count Kafka Connect JMX metric worker
# TYPE kafka_connect_worker_connector_count gauge
kafka_connect_worker_connector_count 3.0

It seems prometheus is not able to scrape the metrics. any specific plugin or some more configuration I need to do at prometheus end?

**Reply by scholzj** [reply]
Right, so in that case you need to check your Prometheus installation and make sure it starts scraping this pod. But I'm afraid that is different depending on how you installed Prometheus etc.

**Reply by yogeshgoel001** [reply]
sure , But I have tried to follow prometheus and grafana installation using https://github.com/strimzi/strimzi-kafka-operator/blob/main/examples/metrics/prometheus-install/
Am I missing something?

**Reply by yogeshgoel001** [reply]
Issue Resolved. Thanks
